dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -145,7 +145,7 @@ valTimeout.desc=Specifies a timeout for validation of pooled connections. When s
 # Vendor properties
 
 properties=Generic JDBC Driver Properties
-properties.desc=List of JDBC vendor properties for the data source. For example, databaseName="dbname" serverName="localhost" portNumber="50000".
+properties.desc=List of JDBC vendor properties for the data source. For example, databaseName="dbname" serverName="localhost" portNumber="50000". This generic properties list can be used when no vendor-specific properties list type is available for the JDBC driver that you are using. Do not specify multiple properties elements under a data source. Instead, place all property name/value pairs on a single properties or properties.{JDBC_VENDOR_TYPE} element.
```

Comment:
Suggested change to simplify wording. Please use "name-value" (hyphen) instead of name/value (slash):

Use this generic properties list when no vendor-specific properties list type is available for your JDBC driver. Do not specify multiple properties elements under a data source. Instead, place all property name-value pairs on a single properties or properties.{JDBC_VENDOR_TYPE} element.

Reply:
Updated as you suggested under commit https://github.com/OpenLiberty/open-liberty/pull/20988/commits/b3c06a9af7afb3ca97e08418a8f962641b5d4ea0